DNN Platform/Website/admin/Skins/BreadCrumb.ascx.cs

```diff
@@ -172,21 +178,22 @@ protected override void OnLoad(EventArgs e)
                     tabUrl = this._navigationManager.NavigateURL(tab.TabID, string.Empty, "GroupId=" + this.GroupId);
                 }
 
-                // Begin breadcrumb
-                this._breadcrumb.Append("<span itemprop=\"itemListElement\" itemscope itemtype=\"http://schema.org/ListItem\">");
-
-                // Is this tab disabled? If so, only render the text
+                // Is this tab disabled? If so, only render a span
                 if (tab.DisableLink)
-                {
-                    this._breadcrumb.Append("<span class=\"" + this._cssClass + "\" itemprop=\"name\">" + tabName + "</span>");
+                {
+                    if (this.LegacyMode)
+                        this._breadcrumb.Append("<span><span class=\"" + this._cssClass + "\">" + tabName + "</span></span>");
+                    else
+                        this._breadcrumb.Append("<span class=\"" + this._cssClass + "\">" + tabName + "</span>");
```

Comment:
I think `CleanerMarkup` or something like that might make more sense than `LegacyMode` in this context.

```suggestion
                    if (this.CleanerMarkup)
                    {
                        this._breadcrumb.Append("<span class=\"" + this._cssClass + "\">" + tabName + "</span>");
                    }
                    else
                    {
                        this._breadcrumb.Append("<span><span class=\"" + this._cssClass + "\">" + tabName + "</span></span>");
                    }
```

Reply:
I agree!